media/materials/programs/camera_lens_flare_fs.glsl

```diff
@@ -30,11 +20,8 @@ vec3 lensflare(vec2 uv,vec2 pos)
 
   float ang = atan(main.y, main.x);
   float dist = length(main); dist = pow(dist,.1);
-  float n = noise(vec2((ang-time/9.0)*16.0,dist*32.0));
 
-  float f0 = 1.0/(length(uv-pos)*16.0/scale+1.0);
-
-  f0 = f0+f0*(sin((ang+time/18.0 + noise(abs(ang)+n/2.0)*2.0)*12.0)*.1+dist*.1+.8);
+  float f0 = 2.0/(length(uv-pos)*16.0/scale+1.0);
```

Comment:
Changed 1.0 to 2.0 here. After removing the legacy animated effect (which was causing ugly visual artifacts) and cc() (which had almost no effect on color), I boosted this number to restore the LensFlare's original appearance.

In my opinion, we should not do this because it makes the LensFlare look washed out and maintains a very big hotspot over the light source. Those features would not never have been there if the broken animated effect had never been included.

Reply:
so do you recommend `1.0` or `2.0` here?

Reply:
I prefer 1.0 because the legacy animation effect made the whole lens flare look overdone. I think a more realistic lens flare effect would have excluded it.